api/db/changelog/db.changelog-20-disable-users.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenstein" id="changelog-20">
+    <addColumn tableName="user">
+      <column name="disabled" type="boolean" defaultValueBoolean="false">
```

Comment:
I would add a timestamp indicating when the account was disabled here, and possibly another user ID indicating who did the disabling, and/or an enum indicated why this account was disabled. (This would be important if we ever intend to support somebody other than the end user disabling their account.)

Reply:
I will add who did the disabling and when it was disabled. That seems good to me.